awx/main/utils/external_logging.py

```diff
@@ -115,6 +116,7 @@ def escape_quotes(x):
     return tmpl
 
 
+@task(queue='rsyslog_configurer')
```

Comment:
But then dispatcher also needs to know to listen on this queue, right? @AlanCoding

Reply:
Yes, I wasn't sure if that was getting added here, or done separately. It is a corresponding change that needs to go with these changes.

Reply:
Added the channel to the dispatcher queues